### PR TITLE
404

### PR DESCRIPTION
**Pull request by BKFighter**

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -1,3 +1,4 @@
+ErrorDocument 404 /views/pages/404.html
```
Comment (BKFighter) [inline .htaccess:1]: I added this line.
(THERE ARE YOU HAPPY GITHUB!!!! ARE YOU HAPPY THAT I MADE A COMMENT ON THE SINGLE LINE THAT I CHANGED!!!!!!!! WILL YOU MAKE THE NOTIFICATION GO AWAY FROM THE STUPID BOTTOM OF THE STUPID PAGE NOW GITHUB!!!!!!!!!!!!!!!!!!! ANSWER ME!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!)

**Comment by ghost** [comment]
I was thinking that maybe we should redirect to the 404 page because it discourages people from reloading the page over and over again

**Comment by haylinmoore** [comment]
why would they want to reload over and over again?

**Comment by BKFighter** [comment]
Because some people would interpret 'some time' to be 4-5 seconds, and then 1-2 seconds, and then as fast as they can mash the reload button...

**Comment by ghost** [comment]
No I meant instead of showing a 404 while on that URL it would redirect you to newawe.com/404.php or something

**Pull request merged**